client/hosting/overview/controller.tsx

```diff
@@ -32,16 +35,34 @@ export function hostingConfiguration( context: PageJSContext, next: () => void )
 			removeQueryArgs( window.location.href, 'hosting_features' )
 		);
 	}
-	context.primary = (
+	const isRemoveDuplicateViewsExperimentEnabled =
+		await getIsRemoveDuplicateViewsExperimentEnabled();
+	context.primary = isRemoveDuplicateViewsExperimentEnabled ? (
+		<PanelWithSidebar>
+			<SettingsSidebar />
+			<div className="hosting-configuration">
+				<Hosting />
```

Comment:
I believe we need to remove `wide` from this line in this case:

https://github.com/Automattic/wp-calypso/blob/5ce861f1e1fddefd08b0548469ac4c7528c00fb6/client/hosting/server-settings/main.tsx#L320

It's because without the two-column card layout, it looks too wide:

|Current|After removing `wide`|
|-|-|
|<img width="1348" alt="image" src="https://github.com/user-attachments/assets/88de4466-91a5-4994-83a7-c87dccfcd874" />|<img width="1370" alt="image" src="https://github.com/user-attachments/assets/8ad3231f-a112-44a0-8a7d-23ed57bca5da" />|

Reply:
I was coming here to say the same thing. :)

Let's use the narrower width for cards across both the General and Server tabs. At the moment Server settings is too wide.

<img width="1073" alt="Screenshot 2025-01-29 at 10 02 37" src="https://github.com/user-attachments/assets/a8a4d409-a096-468d-92b7-44b2735b5dea" />

<img width="1183" alt="Screenshot 2025-01-29 at 10 02 44" src="https://github.com/user-attachments/assets/8c9dbec1-bb8d-4b3a-a0a3-10345955ba38" />

Reply:
Addressed this in https://github.com/Automattic/wp-calypso/pull/98875/commits/8280e6afa0f4e51167853f8d555bb4a492f4cc98!